integrations/service-map/assets/mapping/jaeger-traces.mapping

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
I wonder if there's a way we could start doing automated testing on mapping files. There's a lot of them and they have been inaccurate before.